### PR TITLE
docs: Add documentation for the CLI script

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- Adds `firebase` subfolder for images in the assets folder
- Removes `firebase:download:data` script and adds `cli` on the npm_scripts page
- Adds documentation for using the CLI script on the Firebase page

Corresponds with [this](https://github.com/brown-ccv/honeycomb/pull/275) Honeycomb PR

### CODE REVIEW

File: docs/firebase.mdx [hunk 1]
```diff
@@ -149,8 +149,7 @@ npm run dev:firebase
 npm run firebase:emulators:start
 ```
 
-Honeycomb is now running in the browser and connected to data on an emulated instance of Firestore.
-That data can be viewed at [http://localhost:4000/firestore](http://localhost:4000/firestore).
+Honeycomb is now running in the browser and connected to data on an emulated instance of Firestore. It may be viewed at [http://localhost:4000/firestore](http://localhost:4000/firestore).
```
Comment (tdivoll) [inline docs/firebase.mdx:152]: Does this link always work or do people need something running on localhost to be able to click the link?

File: docs/firebase.mdx [hunk 2]
```diff
@@ -210,10 +209,12 @@ A service account has total administrative access to ts Firebase project. The fi
 npm run cli
 ```
 
-1. Select whether you would like to download or delete data
-2. Enter the ID of a given study
-3. Enter the ID of a given participant on that study
-4. Select the sessions to download/delete
+The CLI script will guide you through the steps needed to manager your data appropriately:
```
Comment (tdivoll) [inline docs/firebase.mdx]:
```suggestion
The CLI script will guide you through the steps needed to manage your data appropriately:

```

**Review by tdivoll — CHANGES_REQUESTED** [review]
Looks good, just one minor cleanup and a question for clarification about the localhost.

File: docs/firebase.mdx (hunk 1 above)
Comment (RobertGemmaJr) [reply docs/firebase.mdx:152]: They need their project to be running locally. Should I remove it as a link then?

Comment (tdivoll) [reply docs/firebase.mdx:152]: Looking at it again, this comes right after they do `nmp run` so it should be intuitive to the users.

**Review by tdivoll — APPROVED** [review]

File: docs/firebase.mdx (hunk 1 above)
Comment (RobertGemmaJr) [reply docs/firebase.mdx:152]: Sweet!

**Pull request merged**